### ISSUE TITLE
While generating from Oracle database relation error occurs, but still states model classes created.

### ISSUE THREAD

**Issue by spartiate480** [Kononnable/typeorm-model-generator#252]
This was the command I ran to get this output:
`typeorm-model-generator -h {$hostname:$port/$service_name} -u {$user} -x {$pass} -e oracle -o .`

```
Relation between tables WORK_LOCATION_TYPE and APPLICATION_USER wasn't found in entity model.
Error occurred in typeorm-model-generator.
typeorm-model-generator@0.4.0  node@v11.3.0
If you think this is a bug please open an issue including this log on https://github.com/Kononnable/typeorm-model-generator/issues
Error
    at Object.LogError (/usr/lib/node_modules/typeorm-model-generator/dist/src/Utils.js:13:21)
    at GetRelations.relationKeys.forEach.relationId (/usr/lib/node_modules/typeorm-model-generator/dist/src/drivers/OracleDriver.js:221:27)
    at Set.forEach (<anonymous>)
    at OracleDriver.GetRelations (/usr/lib/node_modules/typeorm-model-generator/dist/src/drivers/OracleDriver.js:216:22)
Tables not found in selected database. Skipping creation of typeorm model.
Error occurred in typeorm-model-generator.
typeorm-model-generator@0.4.0  node@v11.3.0
If you think this is a bug please open an issue including this log on https://github.com/Kononnable/typeorm-model-generator/issues
[11:02:15 AM] Typeorm model classes created.

```

**Comment by Kononnable** [comment]
Could you paste your schema(DDL statements)? Without them I won't be able to reproduce the issue.

As for generating models after error - it's intentional. It means that there was a problem with that relation and it was skipped entirely or generated code might be missing some of parts of information about this relation and code might syntax throw errors during compilation. If you fix such code manually(or just remove broken code) model should be valid.

**Comment by spartiate480** [comment]
What format should I post them here? I can get an export file straight from my DB or paste the DDL export here in this thread? Whichever is easier...

**Comment by spartiate480** [comment]
Okay, so here is the ddl statements for that table:

`--------------------------------------------------------
--  File created - Monday-March-09-2020   
--------------------------------------------------------
--------------------------------------------------------
--  DDL for Table WORK_LOCATION_TYPE
--------------------------------------------------------

  CREATE TABLE "OMS"."WORK_LOCATION_TYPE" 
   (	"WORK_LOCATION_TYPE_ID" NUMBER, 
	"WORK_LOCATION_DESC" VARCHAR2(100 BYTE), 
	"APPLICATION_USER_ID" NUMBER, 
	"LOGGING_TIMESTAMP" TIMESTAMP (6) DEFAULT systimestamp
   ) SEGMENT CREATION IMMEDIATE 
  PCTFREE 10 PCTUSED 40 INITRANS 1 MAXTRANS 255 NOCOMPRESS LOGGING
  STORAGE(INITIAL 1048576 NEXT 1048576 MINEXTENTS 1 MAXEXTENTS 2147483645
  PCTINCREASE 0 FREELISTS 1 FREELIST GROUPS 1 BUFFER_POOL DEFAULT FLASH_CACHE DEFAULT CELL_FLASH_CACHE DEFAULT)
  TABLESPACE "OMS_DATA" ;
--------------------------------------------------------
--  DDL for Table APPLICATION_USER
--------------------------------------------------------

  CREATE TABLE "OMS"."APPLICATION_USER" 
   (	"APPLICATION_USER_ID" NUMBER(10,0) DEFAULT -1, 
	"USERNAME" VARCHAR2(50 BYTE), 
	"PASSWORD" VARCHAR2(50 BYTE), 
	"ENABLE_FLAG" CHAR(1 BYTE), 
	"CREATE_DATE" DATE DEFAULT sysdate, 
	"LOGGING_TIMESTAMP" TIMESTAMP (6) DEFAULT systimestamp, 
	"USER_START_TAB" VARCHAR2(100 BYTE)
   ) SEGMENT CREATION IMMEDIATE 
  PCTFREE 10 PCTUSED 40 INITRANS 1 MAXTRANS 255 NOCOMPRESS LOGGING
  STORAGE(INITIAL 1048576 NEXT 1048576 MINEXTENTS 1 MAXEXTENTS 2147483645
  PCTINCREASE 0 FREELISTS 1 FREELIST GROUPS 1 BUFFER_POOL DEFAULT FLASH_CACHE DEFAULT CELL_FLASH_CACHE DEFAULT)
  TABLESPACE "OMS_DATA" ;
REM INSERTING into OMS.WORK_LOCATION_TYPE
SET DEFINE OFF;
Insert into OMS.WORK_LOCATION_TYPE (WORK_LOCATION_TYPE_ID,WORK_LOCATION_DESC,APPLICATION_USER_ID,LOGGING_TIMESTAMP) values (101,'Equipment',-1,to_timestamp('08-FEB-13 04.02.04.961165000 PM','DD-MON-RR HH.MI.SSXFF AM'));
Insert into OMS.WORK_LOCATION_TYPE (WORK_LOCATION_TYPE_ID,WORK_LOCATION_DESC,APPLICATION_USER_ID,LOGGING_TIMESTAMP) values (102,'Customer',-1,to_timestamp('08-FEB-13 04.02.04.961165000 PM','DD-MON-RR HH.MI.SSXFF AM'));
--------------------------------------------------------
--  DDL for Index WLT_PK
--------------------------------------------------------

  CREATE UNIQUE INDEX "OMS"."WLT_PK" ON "OMS"."WORK_LOCATION_TYPE" ("WORK_LOCATION_TYPE_ID") 
  PCTFREE 10 INITRANS 2 MAXTRANS 255 COMPUTE STATISTICS 
  STORAGE(INITIAL 1048576 NEXT 1048576 MINEXTENTS 1 MAXEXTENTS 2147483645
  PCTINCREASE 0 FREELISTS 1 FREELIST GROUPS 1 BUFFER_POOL DEFAULT FLASH_CACHE DEFAULT CELL_FLASH_CACHE DEFAULT)
  TABLESPACE "OMS_INDEX" ;
--------------------------------------------------------
--  Constraints for Table WORK_LOCATION_TYPE
--------------------------------------------------------

  ALTER TABLE "OMS"."WORK_LOCATION_TYPE" MODIFY ("LOGGING_TIMESTAMP" NOT NULL ENABLE);
 
  ALTER TABLE "OMS"."WORK_LOCATION_TYPE" ADD CONSTRAINT "WLT_PK" PRIMARY KEY ("WORK_LOCATION_TYPE_ID")
  USING INDEX PCTFREE 10 INITRANS 2 MAXTRANS 255 COMPUTE STATISTICS 
  STORAGE(INITIAL 1048576 NEXT 1048576 MINEXTENTS 1 MAXEXTENTS 2147483645
  PCTINCREASE 0 FREELISTS 1 FREELIST GROUPS 1 BUFFER_POOL DEFAULT FLASH_CACHE DEFAULT CELL_FLASH_CACHE DEFAULT)
  TABLESPACE "OMS_INDEX"  ENABLE;
--------------------------------------------------------
--  Ref Constraints for Table WORK_LOCATION_TYPE
--------------------------------------------------------

  ALTER TABLE "OMS"."WORK_LOCATION_TYPE" ADD CONSTRAINT "WLT_AU_FK" FOREIGN KEY ("APPLICATION_USER_ID")
	  REFERENCES "OMS"."APPLICATION_USER" ("APPLICATION_USER_ID") ENABLE;`

**Comment by spartiate480** [comment]
Added as attachment the ddl statements:

[export.txt](https://github.com/Kononnable/typeorm-model-generator/files/4319731/export.txt)

**Comment by Kononnable** [comment]
Fixed and released in `typeorm-model-generator@0.4.1`.
